inference-engine/src/transformations/src/transformations/op_conversions/convert_interpolate1_to_interpolate4.cpp

```diff
@@ -46,20 +45,7 @@ ngraph::pass::ConvertInterpolate1ToInterpolate4::ConvertInterpolate1ToInterpolat
         if (attrsV0.mode == "nearest") {
             attrsV4.mode = ngraph::opset4::Interpolate::InterpolateMode::nearest;
         } else if (attrsV0.mode == "linear") {
-            // If we write only
-            //    attrsV4.mode = ngraph::op::v4::Interpolate::InterpolateMode::linear;
-            // instead of a conditional statements below when attrsV0.mode == "linear",
-            // then we have a performance drop, because CPU and GPU have no optimized
-            // version of the 'linear' mode.
-            // TODO: delete this conditional statement, when CPU and GPU will have
-            // optimized version of the 'linear' mode.
-            if (input_shape_rank < 5) {
-                attrsV4.mode = ngraph::op::v4::Interpolate::InterpolateMode::linear_onnx;
-            } else if (input_shape_rank == 5) {
-                attrsV4.mode = ngraph::op::v4::Interpolate::InterpolateMode::linear;
```

Comment:
This WA can't be removed until all plugins have required optimizations.

Reply:
Oh, in that case, what would be correct choice for this?
1) Leave it as-is. GPU caffe_bilinear optimization won't be visible to customer until CPU is ready.
2) Fork this pass to a gpu-specific version that allows caffe_bilinear usage.
3) Within this pass, check whether backend is GPU or not. (how can we do this?)

Reply:
I reverted this removal. I will apply if there is any change about this.

Reply:
@vladimir-paramuzov  Currently, optimized caffe bilinear would be NOT selected as this logic is reverted. How about selecting optimized caffe bilinear only if target device is GPU?

Reply:
@byungilm I think we should avoid any device-specific logic in common transformations.

Reply:
If WA is applied, users won't be able to use it. Are you suggesting to merge this commit like that?

Alternatively, we may fork `convert_interpolate1_to_interpolate4` that allows caffe_bilinear and GPU can use the forked function. Not sure which is the right choice as I'm not aware of CPU status..